examples/bpi-monolith-test-suite/bpi/components/zkmock/blockchain/blockchain.js

```diff
@@ -0,0 +1,82 @@
+import { solc } from "solc";
+import { fs } from "fs";
+import { Web3 } from "web3";
+
+export const connectToBlockchain = async () => {
+  if (typeof web3 === "undefined") {
+    var web3 = new Web3(new Web3.givenProvider());
+  } else {
+    var web3 = new Web3(
+      new Web3.providers.HttpProvider("http://localhost:7545")
+    );
+  }
+
+  web3.eth.defaultAccount = web3.eth.accounts[0];
+
+  return web3;
+};
+
+export const compileContract = async (contractName, _contractPath) => {
+  const contractPath = _contractPath;
+  const contractSourceCode = fs.readFileSync(contractPath, "utf8");
+
+  const input = {
+    language: "Solidity",
+    sources: {
+      "test.sol": {
+        content: contractSourceCode,
+      },
+    },
+    settings: {
+      outputSelection: {
+        "*": {
+          "*": ["*"],
+        },
+      },
+    },
+  };
+
+  const inputString = JSON.stringify(input);
+  const output = solc.compile(inputString);
+  const outputJSON = JSON.parse(output);
+  const compiledContract = outputJSON.contracts["test.sol"][contractName];
```

Comment:
what is test.sol?

Reply:
Removed test.sol and updated with the name of the contract file.